### ISSUE TITLE
Cityscapes pre-processing models

### ISSUE THREAD

**Issue by misheeltoli** [fast-vid2vid/fast-vid2vid#4]
Hello,

As mentioned in the paper, semantic maps and instance maps of the cityscapes dataset were pre-processed by **well-trained models.**

Can you please provide what models were used for pre-processing. Where can I download those models?

Thanks in advance.

**Comment by fast-vid2vid** [comment]
Hi, as for Cityscapes dataset, we have used DeepLabV3 network for segmentation maps and Mask R-CNN for instance
segmentation masks.

**Comment by misheeltoli** [comment]
Thank you. 
If possible, can you please share the pre-trained DeepLabV3 and Mask R-CNN models for Cityscapes dataset with 35 classes? I tried to find pre-trained models but all of them were trained with 19 classes. But I need pre-trained models with 35 classes.

Thank in advance.

**Comment by fast-vid2vid** [comment]
Yes, we have re-trained a model with 35 classes for segmentation. I am not sure if I deleted it. I will try to find the pre-trained model.

**Comment by misheeltoli** [comment]
Thank you. Let me know if you find the pre-trained model.

**Comment by fast-vid2vid** [comment]
Hi @misheeltoli , I upload a pretrained DeepLabV3 model with 35 classes on [Google Drive](https://drive.google.com/file/d/19_BGgYUa-rF4x1WB1kWtLvtPlqJXo-R1/view?usp=sharing).

**Comment by misheeltoli** [comment]
@fast-vid2vid, thank you for the DeepLabV3 pre-trained model. Can you also provide the Mask R-CNN pre-trained model, please?

**Comment by fast-vid2vid** [comment]
> @fast-vid2vid, thank you for the DeepLabV3 pre-trained model. Can you also provide the Mask R-CNN pre-trained model, please?

I recommend you use the official pre-trained model.
